micro-rdk/src/common/analog.rs

```diff
@@ -47,7 +47,7 @@ pub struct AnalogResolution {
     pub step_size: f32,
 }
 
-pub trait AnalogReader<Word> {
```

Comment:
why did we end up needing to add Send bound?

Reply:
The modules fail clippy without it. The way we use components (with `Arc<Mutex<Component>>`) is supposed to require that `MoistureSensor` implement `Send`, which stores an `Arc<Mutex<Box<dyn AnalogReader>>`, which in turn requires that anything that implements `AnalogReader` also implement `Send`